PickMeUp/PickMeUp/Model/PMDataManager.h

```diff
@@ -14,6 +14,10 @@ NS_ASSUME_NONNULL_BEGIN
 @interface PMDataManager : NSObject
 @property (nonatomic, strong) NSArray<PMConversation*> *conversations;
 -(void)fillConversations:(nullable void(^)(NSArray<PMConversation *> *))completionBlock;
+-(void)fillDMs:(PMConversation *)convo withBlock:(void(^)(NSArray<PMDirectMessage *> *))block;
```

Comment:
it's worth adding doc to this method and explain how it works, and why completion block might be called twice.